polliwog/plane/_plane_object.py

```diff
@@ -10,34 +10,58 @@
 
 class Plane(object):
     """
-    A 2-D plane in 3-space (not a hyperplane).
+    An immutable 2D plane (not a hyperplane) in 3D space.
 
     Args:
-        point_on_plane (np.arraylike): A reference point on the plane, as a
+        reference_point (np.ndarray): A reference point on the plane, as a
             NumPy array with three coordinates.
-        unit_normal (np.arraylike): The plane normal vector, as a NumPy
-            array with three coordinates.
+        normal (np.ndarray): The plane normal vector, with unit length, as a
+            NumPy array with three coordinates.
+
+    Note:
+        To construct a plane from a non-unit length normal, use
+        `Plane.from_point_and_normal()`.
     """
 
     POSITION_DTYPE = np.float64
     DEFAULT_POSITION_DECIMALS = 6
     DEFAULT_DIRECTION_DECIMALS = 6
```

Comment:
Do these want to be optional arguments in case one might want to change that on the calling side?

Reply:
I added an override parameter to the constructor and a to the factory methods where it felt appropriate.